tests/test_g36_FreezeProtectionStage1.py

```diff
@@ -0,0 +1,91 @@
+import unittest, sys
+import datetime
+
+sys.path.append("./src")
+from lib_unit_test_runner import *
+from library import *
+
+import json
+import pandas as pd
+import numpy as np
+
+
+class TestG36FreezeProtectionStage1(unittest.TestCase):
+    def test_freeze_protection_1_pass(self):
+        points = ["supply_air_temp", "outdoor_damper_command", "outdoor_damper_minimum"]
+
+        timestamp = [
+            datetime(2023, 3, 1, 2, 0, 0),
+            datetime(2023, 3, 1, 2, 5, 2),
+            datetime(2023, 3, 1, 2, 10, 0),
+            datetime(2023, 3, 1, 2, 15, 30),
+            datetime(2023, 3, 1, 2, 18, 0),
+        ]
+
+        data = [[4, 30, 20], [4, 20, 20], [8, 20, 20], [9, 20, 20], [10, 30, 20]]
+        df = pd.DataFrame(data, columns=points, index=timestamp)
+
+        expected_results = pd.Series([True, True, True, True, True])
```

Comment:
Do we not want to test `False` cases? And `untested`? This is a general question for all tests.

Reply:
I generally test all three. Sometimes use one test if all three can be checked at time step level (not very rigorous because the overall flag was not all tested)